sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/icd/IcdHeaderAdapter.java

```diff
@@ -0,0 +1,81 @@
+package org.lfenergy.compas.sct.commons.scl.icd;
+
+import org.lfenergy.compas.scl2007b4.model.TCompasICDHeader;
+import org.lfenergy.compas.scl2007b4.model.TCompasIEDRedundancy;
+import org.lfenergy.compas.scl2007b4.model.TCompasIEDType;
+
+import java.math.BigInteger;
+import java.util.Objects;
+
+import static org.lfenergy.compas.sct.commons.util.CommonConstants.*;
+
+/**
+ * Adapter (ou objet du domaine métier représentant un '<compas:ICDHeader'
+ * Attention ne pas insérer de "currentElements" et autres.
+ */
+public record IcdHeaderAdapter(
```

Comment:
- I feel this class belongs more to RQualif than Compas. Could it be in RQualif instead of Compas ?
- "Adapter" may be the correct word, but it is confusing with other the Compas-Sct Adapter classes. IcdHeaderDomain ?
- Javadoc should be in english in Compas

Reply:
- Why in Rqualif ? I explicitly created this class for compas to improve the use of "icdHeaders objects"
and you can see it is used only in compas

- didn't understood your second point put we call this class `IcdHeaderDomain` if you want. I didn't wanted to include another architecture type in the project at first
- ok for the english